server/config-loader.ts

```diff
@@ -43,6 +69,40 @@ export function load(invalidate = false) {
 		}
 	}
 
+	if ('subprocesses' in config) {
```

Comment:
I don't like treating `undefined` differently from "not defined".

```suggestion
	if (config.subprocesses !== undefined) {
```